LEAF_Request_Portal/templates/subindicators.tpl

```diff
@@ -151,7 +157,13 @@
                     return ($('#<!--{$indicator.indicatorID|strip_tags}-->').val().trim() == '');
                 },
                 setRequiredError: function() {
+                    $([document.documentElement, document.body]).animate({
+                        scrollTop: $('#<!--{$indicator.indicatorID|strip_tags}-->_required').offset().top
+                    }, 700);
```

Comment:
Unless I'm wrong, these are all using the same css values.  Definitely need a properly scoped variable to handle them all.

Reply:
👍 Changed per last conversation